README.md

```diff
@@ -19,20 +19,12 @@ For running locally against docker instances of the following services:
 - localstack  
 - run this application independently e.g. in IntelliJ
 
-`docker compose up hmpps-auth hmpps-nomis-prisoner-api localstack`
-
-or 
-
-`docker-compose up hmpps-auth hmpps-nomis-prisoner-api localstack`
+`docker compose up hmpps-auth hmpps-nomis-prisoner-api hmpps-nomis-visit-mapping-api localstack`
```

Comment:
not sure why these are duplicated

Reply:
`docker-compose` v `docker` a bit OTT, so good to remove, we can just mention `docker` going forward and forget about `docker-compose`   (although not sure how you specify a docker compose file with `docker compose`